build.gradle

```diff
@@ -1,20 +1,71 @@
 buildscript {
-  ext.kotlin_version = '1.4.10'
+  ext.versions = [
+    'minSdk': 21,
+    'compileSdk': 30,
+    'targetSdk': 30,
+    'agp': '4.1.1',
+    'lint': '27.1.1',  // = 23.0.0 + agp
+    'kotlin': '1.4.21',
+  ]
+
+  ext.deps = [
+    'kotlin': [
+      'stdlib': [
```

Comment:
Probably can just remove these deps as they're added by default in 1.4.x Kotlin. I'll be surprised if the 7 or 8 deps are actually needed.

Reply:
actually, will need to leave jdk8 here for the lint module, due to this build error: https://pspdfkit.com/blog/2020/how-updating-to-kotlin-14-broke-our-linter-rules/